druid/src/scroll_component.rs

```diff
@@ -92,83 +73,48 @@ impl Default for ScrollbarsState {
 
 impl ScrollbarsState {
     /// true if either scrollbar is currently held down/being dragged
-    fn are_held(&self) -> bool {
+    pub fn are_held(&self) -> bool {
         match self.held {
             BarHeldState::None => false,
             _ => true,
         }
     }
 }
 
-/// A container that scrolls its contents.
-///
-/// This container holds a single child, and uses the wheel to scroll it
-/// when the child's bounds are larger than the viewport.
-///
-/// The child is laid out with completely unconstrained layout bounds.
-pub struct Scroll<T, W> {
-    child: WidgetPod<T, W>,
-    child_size: Size,
-    scroll_offset: Vec2,
-    direction: ScrollDirection,
-    scrollbars: ScrollbarsState,
+#[derive(Debug, Copy, Clone)]
+pub struct ScrollComponent {
+    pub content_size: Size,
+    pub scroll_offset: Vec2,
+    pub scrollbars: ScrollbarsState,
 }
```

Comment:
This should definitely get a doc comment explaining its purpose and how to use it.

Reply:
Yeah, I documented the pub fns but I forgot to give an overview of the usage and purpose besides the mod doc comment which is naturally pretty limited in the information it provides

Reply:
More thorough documentation has been added